modules/ROOT/pages/introduction.adoc

```diff
@@ -285,11 +285,15 @@ For more information about what data is collected, see link:https://neo4j.com/do
 [[versioning]]
 == Versioning
 
-Neo4j uses semantic versioning (link:{semver-uri}[Semantic Versioning Specification 2.0.0]).
-Given a version number `MAJOR.MINOR.PATCH`, the increment is based on:
+2025.01.0 introduces a new calendar versioning (CalVer) for the Neo4j server instead of semantic versioning (SemVer), which was previously used.
+The new versioning format, *YYYY.MM.Patch*, is based on the year and month of the release, for example, 2025.01.0, 2025.02.0, and so on.
+The patch number is incremented for each release within the same month and reset to 0 at the beginning of each month.
```

Comment:
It may be a stupid question, but why at the beginning of each month? Usually we release at the end of month. I don't think there is any connection between patch number and time of month.

Reply:
Good catch! I guess it was the case at some point in time. 😄

Reply:
I took this from the document about versioning.